ParabolInc/parabol
# feat: team members preview in new meeting team selector

Fixes #5604

**Prev PR: #6989**

https://user-images.githubusercontent.com/466991/182911071-a49393f5-192c-43e6-978e-7e0a987551b4.mov

How to test:

- Open new meeting dialog
- Switch between teams with different count of members
- See member avatar previews are displayed correctly and randomized
